lib/matplotlib/tests/test_axes.py

```diff
@@ -2701,6 +2701,24 @@ def test_boxplot_bad_ci_2():
         ax.boxplot([x, x], conf_intervals=[[1, 2], [1]])
 
 
+def test_boxplot_marker_behavior():
+    plt.rcParams['lines.marker'] = 's'
+    plt.rcParams['boxplot.flierprops.marker'] = 'o'
+    plt.rcParams['boxplot.meanprops.marker'] = '^'
+    fig, ax = plt.subplots()
+    test_data = np.arange(100)
+    test_data[-1] = 150  # a flier point
+    bxp_handle = ax.boxplot(test_data, showmeans=True)
+    for bxp_lines in ['whiskers', 'caps', 'boxes', 'medians']:
```

Comment:
> 
> 
> Do you test the kwarg?

@jklymak yes, it gets checked when the line markers get set to `''` irrespective of what the `rcParams['lines.marker']` says (for 'whiskers', 'caps', 'boxes', 'medians').